prebuild/README.md

```diff
@@ -2,9 +2,16 @@
 
 ## Get Debian base boxes
 
+### For the 64 bit box
+
 ```bash
 vagrant box add debian/contrib-jessie64
 ```
+### For the 32 bit box
+
+```bash
+vagrant box add bento/debian-8.6-i386
```

Comment:
I also sent a mail to one guy in Debian who is involved in the Cloud team, who built the Vagrant base boxes you used for the 64 bit version, and proposed a .json for building a 32bit contrib-jessie box, if we prefer to have an official debian image. Though I couldn't test my .json, with their tools, you need a 64 bit system to build the base boxes.